### DISCUSSION TITLE
How to automatically scale up or down all tenant resources

### DISCUSSION THREAD

**Discussion by philodi-dev** [aws-samples/aws-saas-factory-eks-reference-architecture#48] [Q&A]
First of all, thank you for this well architected solution.
I am currently working on the same architecture for one of my projects and I designed the same architecture except that I was having difficulties in deploying the different resources of my tenants; and boom I came across you and realized that it is easily done with AWS Codepipeline and AWS Codebuild.

However, I have a question that I would like your suggestion on.

I would like to know how to perform automatic scaling of different operating resources as over time we may need to increase storage, memory, etc.

Does the provided example already support this feature or we need to add more services to achieve this.

**Comment by ranjith-aws** [comment]
Hi @Phil-Jayz, The example does not provide that. So do you mean something like a Cluster autoscaler that you can trigger based on if you are low on resources?

**Reply by philodi-dev** [reply]
Yes, that's exactly what I mean.
